### PR TITLE
Add "metalsmith-tweet-embed"

### PR DESCRIPTION
**Pull request by scurker**
Adds a plugin for automatically converting twitter status links to embedded twitter statuses

### CODE REVIEW

**Comment by ismay** [comment]
Hi, thanks for the pr! I commented on a missing comma, it’s causing the build to fail.

File: lib/data/plugins.json [hunk 1]
```diff
@@ -1307,6 +1307,12 @@
     "repository": "https://github.com/hellotoby/metalsmith-to-json",
     "description": "Generates JSON files from markdown, preserving filenames and frontmatter."
   },
+  {
+    "name": "Tweet Embed",
+    "icon": "gridlines",
+    "repository": "https://github.com/scurker/metalsmith-tweet-embed",
+    "description": "Converts Twitter status URLS to embedded Twitter statuses"
+  }
```
Comment (Ajedi32) [inline lib/data/plugins.json]: Still missing a comma:

```suggestion
  },
```

Comment (scurker) [reply lib/data/plugins.json]: Thanks! Sorry I missed this.

**Comment by ismay** [comment]
@Ajedi32 I see that we’re using event-stream and it’s causing the build to fail (which makes sense). Is there a version that we can safely use? Then scurker can rebase and it all should be good to go.

**Comment by ismay** [comment]
Actually from the article:

> You should first eliminate the malicious package from your application, which you can do by reverting back to version 3.3.4 of event-stream.

**Comment by ismay** [comment]
@scurker The build should pass now if you rebase on master (after pulling).

**Pull request merged**